### ISSUE TITLE
Unexpected packet received during login process!

### ISSUE THREAD

**Issue by bladbl** [PaperMC/Paper#3677]
### What behaviour is expected:
Normal server login

### What behaviour is observed:
![image](https://user-images.githubusercontent.com/47465708/85959143-46fcc580-b9a3-11ea-998a-3e41e2b6a640.png)

### Steps/models to reproduce:
The problem is on build paper #11-13 #17
Everything works on paper-10, also on spigot.

### Plugin list:
![image](https://user-images.githubusercontent.com/47465708/85959233-b1156a80-b9a3-11ea-9b16-bd2657d7381a.png)

### Paper build number:
Waterfall#358 and Paper 11-13
### Anything else:
https://pastebin.com/2Hy9QWfA

Bungee plugins
![image](https://user-images.githubusercontent.com/47465708/85962468-8cc48880-b9b9-11ea-928c-184679c9be41.png)

**Comment by bladbl** [comment]
Fix in paper 27

**Comment by DmitryRendov** [comment]
this is still an issue in paper27 for me, switched to spigot for now...

**Comment by Chronic-Reflexes** [comment]
Same here Happening in paper-43, paper-42

**Comment by Proximyst** [comment]
Please open a new issue with all information available to you.

**Comment by Wolfieheart** [comment]
This appears to be caused by DiscordSRV 1.19.1 SNAPSHOT - As Disabling it fixes the issue in question. 

TL;DR - Its basically caused by DiscordSRV and not by Bungee, Waterfall or Paper :)
